packages/site-admin/src/router/hooks/use-history/index.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { useEvent } from '@wordpress/compose';
```

Comment:
`useEvent` is not available on `@wordpress/compose` until 7.8.0, see [changelog here](https://github.com/WordPress/gutenberg/blob/trunk/packages/compose/CHANGELOG.md).

`packages.json` declares `"@wordpress/compose": "^7.18.0",`